front/pages/api/w/[wId]/invitations/index.ts

```diff
@@ -152,7 +158,48 @@ async function handler(
         });
       }
       const existingMembers = await getMembers(auth);
-
+      const unconsumedInvitations = await MembershipInvitation.findAll({
+        where: {
+          workspaceId: owner.id,
+          status: ["pending", "revoked"],
```

Comment:
Should we account for `revoked` in there?

Reply:
Yeah because otherwise you can just revoke and re-send. Maybe we should discard revoked ones that are too old ? TBH if you have more than a few revoked it's fishy

Reply:
But what happens when you have revoked 50 invitations, you can't invite anymore?

Reply:
Simplified the whole thing. Now were just looking at invitations from the last 24 hours. IMO it's enough to limit abuse.